all.json

```
@@ -4,6 +4,9 @@
 		"07e96d43-381a-46a3-9c16-6daf97213efc.co",
 		"0vvwvuniswap.top",
 		"0vvwwuniswap.top",
+		"fixtechnicalsupport.com",
+		"dogclinicaveterinaria.itautorectifywalletdap.com",
+		"whitelist.officiav2.online",
```

```suggestion
		"dogclinicaveterinaria.itautorectifywalletdap.com",
		".itautorectifywalletdap.com",
		"whitelist.officiav2.online",
		"officiav2.online",
```

Add top-level domains.

---

What do you mean?